include/zephyr/arch/riscv/csr.h

```diff
@@ -233,4 +234,40 @@
 				: "memory");			\
 })
 
+/* uhartid -- user hardware thread id, seems to be not yet defined in toolchain */
+#define	uarthid		0x014
```

Comment:
nit: should be `uhartid`

Reply:
actually, its bigger than that. `uhartid ` is not a standard CSR

Reply:
agree